core/src/index/cmake/DefineOptionsCore.cmake

```diff
@@ -10,7 +10,14 @@
 # is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 # or implied. See the License for the specific language governing permissions and limitations under the License.
 #-------------------------------------------------------------------------------
+#-------------------------------------------------
+set_option_category("FPGA version")
 
+if (MILVUS_FPGA_VERSION)
+    define_option(KNOWHERE_FPGA_VERSION "Build FPGA version" ON)
+else ()
+    define_option(KNOWHERE_FPGA_VERSION "Build FPGA version" OFF)
+endif ()
```

Comment:
It seems that "KNOWHERE_FPGA_VERSION" is never used.
May I remove it?

Reply:
yes , you can.